cmd/compute-domain-daemon/computedomain.go

```diff
@@ -121,13 +128,43 @@ func (m *ComputeDomainManager) Stop() error {
 	return nil
 }
 
+// Get gets the ComputeDomain by UID from the informer cache.
+func (m *ComputeDomainManager) Get(uid string) (*nvapi.ComputeDomain, error) {
+	objs, err := m.informer.GetIndexer().ByIndex("uid", uid)
+	if err != nil {
+		return nil, fmt.Errorf("error retrieving ComputeDomain by UID: %w", err)
+	}
+	if len(objs) == 0 {
+		return nil, nil
+	}
+	if len(objs) != 1 {
+		return nil, fmt.Errorf("multiple ComputeDomains with the same UID")
```

Comment:
So, this is basically a cache invalidation problem. And here we give up because we can't quite solve that? We don't know which one is latest?